eko-cosmetics

MariiaSuper opened the pull request:
https://MariiaSuper.github.io/Eco_cosmetics/

volodymyr-soltys97 reviewed (changes requested):
Demo link. You need to check and fix it
<img width="913" alt="image" src="https://github.com/mate-academy/Eco_cosmetics/assets/64750958/f167d1b0-784c-4005-bf5b-5c923d247f1a">

mykhalenych reviewed (changes requested):
1. delete autocomplete styles
<img width="447" alt="Screenshot 2023-07-05 at 15 51 06" src="https://github.com/mate-academy/Eco_cosmetics/assets/53228259/dc3776d1-32e6-4d9e-a535-29004a93c9f2">
2. disable scroll when menu is open
https://github.com/mate-academy/Eco_cosmetics/assets/53228259/8e006205-06f4-486b-be78-47d11aaac63e

3. hover works for two icons simultaneously
https://github.com/mate-academy/Eco_cosmetics/assets/53228259/179f1867-4832-49fa-8fc3-e93c85b4d794

anastasiiavorobiova reviewed (changes requested):
Awesome landing! Comments to fix: 
<img width="997" alt="Screenshot 2023-07-05 at 18 18 26" src="https://github.com/mate-academy/Eco_cosmetics/assets/135232001/2e317c49-74bb-47c9-8cb4-8ef57c979384">
Consider using an HTML validator to make your HTML structure valid for browsers. 
<img width="533" alt="Screenshot 2023-07-05 at 18 26 51" src="https://github.com/mate-academy/Eco_cosmetics/assets/135232001/139681f8-4725-4803-a946-d2ef0076918d">
Check font sizes here for desktop
<img width="446" alt="Screenshot 2023-07-05 at 18 25 52" src="https://github.com/mate-academy/Eco_cosmetics/assets/135232001/b888fd89-d1fe-47f4-a3e9-da7438d4460d">
Styles for this field don't match the Figma design
<img width="537" alt="Screenshot 2023-07-05 at 18 25 26" src="https://github.com/mate-academy/Eco_cosmetics/assets/135232001/f744648a-753c-43c6-ac84-e7efcfef6fdf">
Content flickering happens on hover, consider fixing it

vlad-demchuk reviewed (changes requested):
GJ

just small fix
add cursor pointer to these buttons
<img width="309" alt="image" src="https://github.com/mate-academy/Eco_cosmetics/assets/102411130/d1c0cd2a-d8b3-4197-8e67-3564cbc95da1">
<img width="368" alt="image" src="https://github.com/mate-academy/Eco_cosmetics/assets/102411130/5217f1f6-19e3-4bb2-8cc5-94249f28841c">

VitaliyBondarenko1982 reviewed (changes requested):
Good job.
Almost done.
Found one more small bug
![chrome-capture-2023-6-6 (1)](https://github.com/mate-academy/Eco_cosmetics/assets/33029864/371ef645-7483-484e-b3f1-bc43b4e11b3d)
for fix it add `resize: none` for textarea styles.

volodymyr-soltys97 reviewed (changes requested):
1. Need to fix
Demo
<img width="356" alt="image" src="https://github.com/mate-academy/Eco_cosmetics/assets/64750958/28f3199b-5d96-497c-afcc-1e5fa941e674">

Design
<img width="484" alt="image" src="https://github.com/mate-academy/Eco_cosmetics/assets/64750958/dac40bd2-8060-4b3e-be27-099bf29b7d04">

2. Add `cursor: pointer` for all clickable elements
<img width="482" alt="image" src="https://github.com/mate-academy/Eco_cosmetics/assets/64750958/6bd4131d-e1fa-4d26-a838-f7fbe9dc8fda">

anastasiiavorobiova commented on src/index.html:
<img width="925" alt="Screenshot 2023-07-06 at 15 40 20" src="https://github.com/mate-academy/Eco_cosmetics/assets/135232001/18cb9851-94fd-4d23-ae57-08f5633c13b2">
Consider fixing validation errors

MariiaSuper replied:
I corrected what you told me... I don't see anything else

anastasiiavorobiova reviewed (changes requested):
Great work! Unfortunately, there are a lot of things that are not fixed from previous reviews

anastasiiavorobiova reviewed (changes requested):
Great solution! Needs to fix the last two comments: 
<img width="615" alt="Screenshot 2023-07-06 at 18 21 25" src="https://github.com/mate-academy/Eco_cosmetics/assets/135232001/8467f45d-c1b0-47d1-be6f-524169a87db1">
<img width="659" alt="Screenshot 2023-07-06 at 18 21 54" src="https://github.com/mate-academy/Eco_cosmetics/assets/135232001/88a04651-5192-471b-a5b0-4aff23252a22">
Feel free to ask for help in the chat

MariiaSuper commented:
> Great solution! Needs to fix the last two comments: <img alt="Screenshot 2023-07-06 at 18 21 25" width="615" src="https://user-images.githubusercontent.com/135232001/251496770-8467f45d-c1b0-47d1-be6f-524169a87db1.png"> <img alt="Screenshot 2023-07-06 at 18 21 54" width="659" src="https://user-images.githubusercontent.com/135232001/251496784-88a04651-5192-471b-a5b0-4aff23252a22.png"> Feel free to ask for help in the chat

added a gap of 51px (as in the design), changed the colors of the menu in the block
